extensions/wikibase/tests/src/org/openrefine/wikibase/testing/WikidataRefineTest.java

```diff
@@ -29,10 +36,18 @@ public class WikidataRefineTest {
     private List<Project> projects = new ArrayList<Project>();
     private List<ImportingJob> importingJobs = new ArrayList<ImportingJob>();
 
+    /**
+     * @deprecated use {@link #createProject(String[], Serializable[][])}
+     */
+    @Deprecated
     public Project createCSVProject(String input) {
         return createCSVProject("test project", input);
     }
```

Comment:
This duplication with `RefineTest.java` will be addressed by the later modularization, isolating test utilities in an independent Maven module that extensions (be they in this repository or elsewhere) can depend on, and avoid having to reimplement our test utilities to write their own tests.